<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>transferwindow</class>
 <widget class="QDialog" name="transferwindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog{
	background-color: rgb(255, 178, 25)
}</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>0</y>
     <width>181</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
	color:black;
}</string>
   </property>
   <property name="text">
    <string>Transfer Window</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="AccNumSelect">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>190</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>190</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
	color: black;
}</string>
   </property>
   <property name="text">
    <string>To Account Number:</string>
   </property>
  </widget>
  <widget class="QLabel" name="UsernamLabel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>90</y>
     <width>481</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    color: black;
}</string>
   </property>
   <property name="text">
    <string>Username:  </string>
   </property>
  </widget>
  <widget class="QLabel" name="BalanceLabel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>120</y>
     <width>481</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    color: black;
}</string>
   </property>
   <property name="text">
    <string>Balance:  </string>
   </property>
  </widget>
  <widget class="QLabel" name="AccNumLabel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>60</y>
     <width>481</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    color: black;
}</string>
   </property>
   <property name="text">
    <string>Account Number: </string>
   </property>
  </widget>
  <widget class="QLabel" name="StatusLabel">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>170</y>
     <width>241</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
	color: black;
}</string>
   </property>
   <property name="text">
    <string>Status:
</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>300</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
	color: black;
}</string>
   </property>
   <property name="text">
    <string>Amount:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="AmountSelect">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>300</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="TransferBtn">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>300</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	background-color: white;
	color: black;
}</string>
   </property>
   <property name="text">
    <string>Transfer</string>
   </property>
  </widget>
  <widget class="QPushButton" name="HomeBtn">
   <property name="geometry">
    <rect>
     <x>601</x>
     <y>453</y>
     <width>80</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	background-color: white;
	color: black;
}</string>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
